integration/web-include.js

```diff
@@ -0,0 +1,3 @@
+import fledge from '../src/index.js';
+
+window.fledge = fledge;
```

Comment:
any reason we're dropping this on the `window` object?

Reply:
I need to be able to access it from puppeteer, this seemed like the cleanest way. I'm open to other suggestions.